hack/release.sh

```diff
@@ -295,18 +295,10 @@ EOF
 	fi
```

Comment:
@tiborvass yes, I'm working on a separate branch to remove all rpm/deb/release related stuff. There's still some issues around that, so decided to keep this PR just for the `install.sh`